### PR TITLE
DEV mariateo embeded Youtube video

### PR DESCRIPTION
**Pull request by MariaTeo**

### CODE REVIEW

**Review by biggicode — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]
✔

File: src/_component/section-title/section-title.pug [hunk 1]
```diff
@@ -2,6 +2,12 @@
   .section__title-wrapper
     .section__title=sectionTitle
 .column(span="row" height="10")
+.column(span="row")
+  iframe(src="https://www.youtube.com/embed/G7RgN9ijwE4"  title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen)
```
Comment (marianzburlea) [inline src/_component/section-title/section-title.pug]: separate attributes on multiple lines
```
iframe(
  src="https://www.youtube.com/embed/G7RgN9ijwE4"
  title="YouTube video player"
  frameborder="0"
  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
  allowfullscreen
)
```

**Review by marianzburlea — CHANGES_REQUESTED** [review]
make the youtube button work

**Review by adelacecilia — APPROVED** [review]
Nice :)

**Review by biggicode — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]

**Review by Adrian01C — APPROVED** [review]
niceee

**Review by RalucaRbc — APPROVED** [review]
💯

**Review by simonamarcu2 — APPROVED** [review]
Nice work!

File: src/_component/section-title/section-title.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+.youtube {
+  display: none;
+  transition: 0.5s ease-out;
+  flex-direction: column;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.85);
+  width: 100%;
+  height: 100%;
+  z-index: 1;
+  transform: translate(-50%,-50%) scale();
```
Comment (marianzburlea) [inline src/_component/section-title/section-title.scss:24]: Add extra space and remove the scale if you're not using it.
`transform: translate(-50%, -50%);`

File: src/_component/section-title/section-title.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  color: white;
+}
+
+.popup.active .overlay {
+  display: block;
+}
+
+.popup.active .content__buton {
+  transform: all 300ms ease-in-out;
+  transform: translate(-50%,-50%) scale(1);
```
Comment (marianzburlea) [inline src/_component/section-title/section-title.scss:66]: Add extra space
`  transform: translate(-50%, -50%) scale(1);`

File: src/index.pug [hunk 4]
```diff
@@ -5,6 +5,17 @@ html
...
   body 
+    input.youtube__checkbox(type="checkbox", id="popup")
+    .youtube 
+      .buton__inchis 
+        label(for="popup")
+          i.icon__cross 
+      .content__buton
+        iframe(
+          src="https://www.youtube.com/embed/G7RgN9ijwE4"  
+          title="YouTube video player" frameborder="0"
```
Comment (marianzburlea) [inline src/index.pug:16]: press <kbd>Enter</kbd> beofer frameborder:

```
title="YouTube video player"
frameborder="0" 
```

File: src/index.pug [hunk 5]
```diff
@@ -5,6 +5,17 @@ html
...
+    input.youtube__checkbox(type="checkbox", id="popup")
+    .youtube 
+      .buton__inchis 
+        label(for="popup")
+          i.icon__cross 
+      .content__buton
+        iframe(
+          src="https://www.youtube.com/embed/G7RgN9ijwE4"  
+          title="YouTube video player" frameborder="0" 
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen
```
Comment (marianzburlea) [inline src/index.pug:17]: same for `allowfullscreen`

**Review by marianzburlea — APPROVED** [review]
💯

**Comment by adelacecilia** [comment]
Looks nice! :D

**Pull request merged**